2019-06-17T20:56:20|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:20|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:20|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:20|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:21|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:22|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:23|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:24|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:25|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:26|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:27|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:28|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:29|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:30|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:31|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:31|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:31|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:31|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:31|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:31|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:31|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:32|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:32|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:32|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(*avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:32|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:32|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T20:56:32|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: format() argument after * must be an iterable, not numpy.float64. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 144, in update_display\n    title += \'<b>\\u03BC</b> = {:3.0f} \'.format(avg_val[i])\n', 'TypeError: format() argument after * must be an iterable, not numpy.float64\n']
2019-06-17T21:01:45|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:45|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:45|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:45|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:45|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:46|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:46|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:46|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:46|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:46|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:46|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:47|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:47|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:47|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:47|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:47|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:48|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:48|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:48|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:48|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:48|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:50|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    title += \'<b>\\u03BC</b><sub>{}</sub> = {:3.0f} \'.format(rate,avg_val[i])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:01:50|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: name 'avg_val' is not defined. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 140, in update_display\n    self.plotlines[rate].setVisible(self.settings[\'{}_visible\'.format(rate)])\n', "NameError: name 'avg_val' is not defined\n"]
2019-06-17T21:05:23|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundry\measurement.py", line 25, in run
    self.measurement._thread_run()
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundry\measurement.py", line 183, in _thread_run
    self.run()
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundryHW\picoquant\hydraharp_optimizer.py", line 113, in run
    self.optimize_history[self.optimize_ii,:] = [hh_hw.settings[x] for x in self.rates.keys()]
AttributeError: 'list' object has no attribute 'keys'

2019-06-17T21:05:24|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundry\measurement.py", line 25, in run
    self.measurement._thread_run()
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundry\measurement.py", line 183, in _thread_run
    self.run()
  File "C:\Users\lab\Documents\foundry_scope\ScopeFoundryHW\picoquant\hydraharp_optimizer.py", line 113, in run
    self.optimize_history[self.optimize_ii,:] = [hh_hw.settings[x] for x in self.rates.keys()]
AttributeError: 'list' object has no attribute 'keys'

2019-06-17T21:08:48|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:48|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:49|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:50|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:50|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:50|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
2019-06-17T21:08:50|ERROR|HydraHarpOptimizerMeasure|hydraharp_channel_optimizer Failed to update figure1: 'avg_len'. ['Traceback (most recent call last):\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\measurement.py", line 279, in _on_display_update_timer\n    self.update_display()\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundryHW\\picoquant\\hydraharp_optimizer.py", line 133, in update_display\n    N_AVG = min(self.settings[\'avg_len\'],self.settings[\'history_len\']-1)\n', '  File "C:\\Users\\lab\\Documents\\foundry_scope\\ScopeFoundry\\logged_quantity.py", line 1325, in __getitem__\n    return self._logged_quantities[key].val\n', "KeyError: 'avg_len'\n"]
